modules/installation-gcp-permissions.adoc

```diff
@@ -23,32 +24,64 @@ install {product-title}. To deploy an {product-title} cluster, the service
 account requires the following permissions. If you deploy your cluster into an existing VPC, the service account does not require certain networking permissions, which are noted in the following lists:
 
 .Required roles for the installation program
-* Compute Admin
-* Security Admin
-* Service Account Admin
-* Service Account User
-* Storage Admin
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|Compute Admin
+|Gives full control of all compute resources. This role is needed by the service account to manage virtual machine instances. {product-title} is a self-managed platform that requires permission to create cloud infrastructure to automatically scale and failover during normal operation.
+
+|Security Admin
+|Allow the service account to create, modify, and delete firewall rules and SSL certificates within the GCP project.
+
+|Service Account Admin
+|Creates and manages service accounts.
+
+|Service Account User
+|Runs operations as the service account.
+
+|Storage Admin
+|Grants full control of storage objects and buckets, and is needed to help provide persistent storage to the cluster.
+|===
 
 .Required roles for creating network resources during installation
-* DNS Administrator
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|DNS Administrator
```

Comment:
I believe the cluster also needs this access to add new DNS entries for ingress operator via the kube could integration.